posthog/api/test/test_signup.py

```diff
@@ -690,6 +691,35 @@ def test_social_signup(self, *args):
         assert user.is_active is True
         assert user.is_staff is True
 
+    def test_social_login_give_staff_privileges(self, *args):
+        # This creates a user with is_staff=False
+        User.objects.bootstrap("Tech R Us", "charlie@tech-r-us.com", None, first_name="Charlie")
```

Comment:
we could replace the comment with `assert user.is_staff is False` after the user bootstrap